schemas/code_signature.yml

```
@@ -123,3 +123,15 @@
       description: >
         Date and time when the code signature was generated and signed.
       example: "2021-01-01T12:10:30Z"
+
+    - name: thumbprint
+      level: extended
+      type: keyword
+      short: Unique ID (sha256 hash) of the certificate .
+      description: >
+        A cryptographic fingerprint that uniquely identifies the certificate.
+
+        This hash is generated by applying the SHA-256 function to the
+        certificate's raw data. It is always lowercase.
```

Shorter for consistency?

```suggestion
      description: >
        Certificate SHA256 hash that uniquely identifies the code signer.
```
https://github.com/elastic/ecs/blob/77bb0b09ab84ec63c7d831269536a83551fb4d69/schemas/hash.yml#L49